airbyte-server/build.gradle

```diff
@@ -18,6 +18,7 @@ dependencies {
     implementation project(':airbyte-config:persistence')
     implementation project(':airbyte-config:init')
     implementation project(':airbyte-db')
+    implementation project(':airbyte-integrations')
```

Comment:
Not really ideal, but it's already a transitive dependency through the scheduler. I could move the integration registry into `airbyte-config` if you feel strongly about this.

Reply:
all good for now. that will change soon anyway since integrations map will become a config